backend/src/main/java/quarano/department/web/TrackedCaseStatusAware.java

```diff
@@ -46,7 +46,10 @@ public static Links getDefaultLinks(TrackedCase trackedCase) {
 				.and(MvcLink.of(controller.getDiaryOfCase(caseId, null), DIARY))
 				.and(MvcLink.of(controller.getCase(caseId, null), IanaLinkRelations.SELF))
 				.andIf(trackedCase.isIndexCase(), MvcLink.of(controller.getContactsOfCase(caseId, null), CONTACTS))
-				.andIf(trackedCase.getStatus().equals(Status.IN_REGISTRATION), () -> MvcLink.of(uri, RENEW))
+				.andIf(trackedCase.getStatus().equals(Status.IN_REGISTRATION) ||
+						((trackedCase.getStatus().equals(Status.REGISTERED) || trackedCase.getStatus().equals(Status.TRACKING))
+								&& trackedCase.getTrackedPerson().getAccount().isEmpty()), () -> MvcLink.of(uri, RENEW))
```

Comment:
Can we simplify the brackets here? The only thing that is relevant is: `trackedCase.getTrackedPerson().getAccount().isEmpty()`. If we also want to check the status to be sure, then it's enough that one of the three is set, right?